app/styles/_kve.less

```diff
@@ -170,7 +170,8 @@
 }
 
 .key-value-editor .key-value-editor-input,
-.key-value-editor-header {
+.key-value-editor-header,
+.environment-from-editor-header {
```

Comment:
@sg00dwin made adjustment for the column widths see PR screenshot. Still looking the column containing the prefix field is a few pixels off